recipes/pybufr-ecmwf/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
+  export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib
```

Comment:
I'm guessing here - but does this explicitly need to link to the python lib? Which it doesn't appear the linking command is?